### ISSUE TITLE
About mask scaling factor

### ISSUE THREAD

**Issue by QiuJueqin** [scut-mingqinchen/self2self#7]
In [Punet.py](https://github.com/scut-mingqinchen/self2self/blob/e376c483d032ad54925da1355be0e01acf4473e1/network/Punet.py#L152), the mask is scaled by a factor that is equal to dropout rate (0.7), is it a bug?

`mask_tensor = tf.nn.dropout(mask_tensor, 0.7) * 0.7`

According to TF doc, the tensor being dropouted has expectation of 1.0/(1-rate), which would be 3.3333 in this setting. So the scaling factor should be 0.3 instead of 0.7?

`mask_tensor = tf.nn.dropout(mask_tensor, 0.7) * 0.3`

**Comment by QiuJueqin** [comment]
This repo used TF1.14, in which the 2nd argument in `tf.nn.dropout` is `keep_prob` instead of `rate` in TF2.x. The author's implementation was correct. 

Issue closed.
